server.go

```diff
@@ -2372,6 +2373,16 @@ func (s *server) outboundPeerConnected(c *connmgr.ConnReq, conn net.Conn) {
 		s.connManager.Disconnect(c.ID())
 		return
 	}
+
+	// Enforce the minimum protocol limit on outbound connections.
```

Comment:
This should be next to the other one in `OnVersion` because this code runs later and it makes more sense to do it alongside the other check.

So, if you move it up there, you'd just need to do:

```Go
if !isInbound && p.ProtocolVersion() < wire.RemoveRejectVersion { ... }
```